src/lib/entry-point-transforms.ts

```diff
@@ -105,9 +105,9 @@ export const transformSources =
     // TODO: doesn't work any more .... path.relative(secondary.basePath, primary.basePath);
     const relativeDestPath: string = path.relative(entryPoint.destinationPath, pkg.primary.destinationPath);
     await writePackage(entryPoint, {
-      main: ensureUnixPath(path.join(relativeDestPath, 'bundles', entryPoint.umdPackageName)), // XX entryPoint.flatModuleFile + '.umd.js')),
-      module: ensureUnixPath(path.join(relativeDestPath, 'esm5', entryPoint.esmPackageName)), // XX entryPoint.flatModuleFile + '.js')),
-      es2015: ensureUnixPath(path.join(relativeDestPath, 'esm2015', entryPoint.esmPackageName)), // XX entryPoint.flatModuleFile + '.js')),
+      main: ensureUnixPath(path.join(relativeDestPath, 'bundles', entryPoint.flatModuleFile + '.umd.js')),
+      module: ensureUnixPath(path.join(relativeDestPath, 'esm5', entryPoint.flatModuleFile + '.js')),
+      es2015: ensureUnixPath(path.join(relativeDestPath, 'esm2015', entryPoint.flatModuleFile + '.js')),
```

Comment:
I'm pretty sure I get how this works. 
I think you centralized the file name to the `flatModuleFile` and can then transform that for each of the package file outputs. 

Thanks for adding this so fast!